client/src/components/code/ai-chat/index.tsx

```
@@ -46,66 +56,226 @@ export const useAIChat = () => {
+                    setMessages(prev => [...prev, { role: "agent", content: completionMessage }]);
+                }
+                setIsChatPending(false);
+                setProgressSteps([]);
+                setRoute(null);
+                routeRef.current = null;
+                pendingProgressRef.current = [];
+                closeStream();
+                break;
+            case "error":
+                setIsChatPending(false);
+                const lastStep = progressStepsRef.current[progressStepsRef.current.length - 1];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)